canopen_402_driver/include/canopen_402_driver/node_interfaces/node_canopen_402_driver_impl.hpp

```diff
@@ -533,14 +465,34 @@ bool NodeCanopen402Driver<NODETYPE>::set_mode_position()
   }
 }
 
+template <class NODETYPE>
+bool NodeCanopen402Driver<NODETYPE>::set_mode_interpolated_position()
```

Comment:
From where Are these methods called? It seems to me that structure of them is repeating. Can we make a general method that gets the mode Type as an argument, which is called from all other places specifying mode?

Even if the methods are callbacks from parent classes, I think there is value to ass one more level of abstraction so they become one-liners.

Reply:
This is a good idea. There is an example in #102 that adds set operation mode to the driver.